pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
```

Comment:
Nit: FMI Regarding your choice for dependency management, see a list of points below that I consider pros for poetry:
- Dependency bloat (as n anaconda)
- Unexpected behaviour of anaconda/miniconda with pip (e.g. package display, package install)
- Better dep conflict handling compared to pip
- automatic logging and updates of package metadata 
Can you give this message a thumbs up if these points cover all pros you had in mind when choosing poetry. Otherwise would you mind listing any additional points and/or cons?

Reply:
I think poetry will still conflict sometimes with anaconda -- this is unavoidable because conda tends to clobber packages and have older "packaged" versions. To those who wish to use anaconda I say "don't", and to those who insist depsite this, I say "good luck" 

> Better dep conflict handling compared to pip

Specifically, poetry uses a proper graph based dependency solver, which pip does not.

Also, the pyproject.toml format is purely declarative and avoids the setup.py problem where a program must be run in order to know about package metadata, which is a primary source of and complaint about things which slow poetry down when package solving. Also, pyproject.toml is actually a pep standard (although poetry itself may not be) -- to see this in action, you can create a virtualenv and activate it and do `pip install .` and this also works as expected (installs the package and its deps).

So, I should probably change the readme back...

Reply:
On 2nd thoughts, I'll leave it since poetry solves the environment properly.

Reply:
LGTM